app.js

```diff
@@ -0,0 +1,112 @@
+const title = document.querySelector('.title-book');
+const author = document.querySelector('.author-name');
+const bookData = [];
+let i = 0;
+
+/* add new book */
+
```

Comment:
- [ ] Kindly fix the following behavior which happens whenever the user adds new books to existing books that are loaded from local storage.

https://user-images.githubusercontent.com/62290545/176191157-c2a91a81-7b7f-4ca5-a6c0-36cd5895dee1.mp4